media/doc/README.WINE

```diff
@@ -49,7 +49,7 @@ reactos/dll/win32/atl                 # Synced to WineStaging-2.9
 reactos/dll/win32/atl80               # Synced to WineStaging-2.9
 reactos/dll/win32/atl100              # Synced to WineStaging-2.9
 reactos/dll/win32/avifil32            # Synced to WineStaging-2.9
-reactos/dll/win32/bcrypt              # Synced to WineStaging-1.9.23
+reactos/dll/win32/bcrypt              # Synced to WineStaging-1.9.23 (+ winetest synced to 8d8936cbb6fea3cac862e059e814527f5361f48b, a.k.a 20161206-BJ)
```

Comment:
This will land in the upcoming sync.

Reply:
I shall wait for that then...

Reply:
`WineSync` has not happened yet for this file. Could this PR be committed then?